meetings-en/meeting-1964.yml

```diff
@@ -16,14 +16,14 @@ resolutions:
   considerations:
   - type: acknowledging
     date_effective: 1964-01-01
-    message: empowered by cgpm-resolution:en/12/5[Resolution 5] of the 12th Conférence
+    message: *empowered* by cgpm-resolution:en/12/5[Resolution 5] of the 12th Conférence
       Générale des Poids et Mesures to name atomic or molecular frequency standards
       for temporary use for time measurements in physics,
   actions:
   - type: decides
     date_effective: 1964-01-01
-    message: that the standard to be employed is the transition between the
+    message: '*declares* that the standard to be employed is the transition between the
```

Comment:
@anermina as per our discussion, let's change the `type: decides` to `type: declares`. I've updated https://github.com/metanorma/resolution-model/commit/0c62041df08e655e16824bccb8dd09166e78d3c2 accordingly. Thanks!

Reply:
`type: declares` was quite frequently used in both CGPM and CIPM resolutions. Changes are applied for the files in both repositories. However, there are also some less frequently used keywords which appear in these resolutions, such as abrogates, empowers, reaffirms, etc. Should we add them as well?